azure-pipelines/build-pr.yml

```diff
@@ -31,21 +31,21 @@ jobs:
 
   # Run jobs that only build the environment. These jobs have a high chance of succeeding and filling the build
   # cache. Pytest, etc legs will only fill the cache if they succeed.
-  - job: CreateCondaEnvCache_Windows
-    pool:
-      vmImage: 'windows-2019'
-    steps:
-      - template: inner_eye_env.yml
+  # - job: CreateCondaEnvCache_Windows
```

Comment:
It would be cleaner to remove this, it's easy to add in if we need it again in the future.